src/cunumeric/execution_policy/execution_policy_helpers.h

```diff
@@ -0,0 +1,11 @@
+#ifndef SRC_CUNUMERIC_EXECUTION_POLICY_EXECUTION_POLICY_HELPERS_H_
+#define SRC_CUNUMERIC_EXECUTION_POLICY_EXECUTION_POLICY_HELPERS_H_
+
+// If nvcc or clang indicates CUDA compilation
+#if defined(__CUDACC__) || defined(__CUDA__)
+#define CUDA_FUNCTION __host__ __device__
```

Comment:
There already exists an equivalent macro: https://github.com/nv-legate/cunumeric/blob/06821fdbeef9f525b8bbcac7f38462cdd38e08dc/src/cunumeric/unary/unary_op_util.h#L273 can we use it instead?

Reply:
Yep, thank you, I will use that one.